setup.py

```
@@ -0,0 +1,333 @@
+                f"WARNING: cargo not found, skipping build of {binary_name}",
+                file=sys.stderr,
+            )
+            return
+
+        cmd = ["cargo", "build", "--release", "--bin", binary_name]
+        if manifest_path:
+            cmd.extend(["--manifest-path", manifest_path])
+
+        print(f"Building {binary_name}...")
+        try:
+            subprocess.run(cmd, check=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)